lxd/storage/drivers/driver_pure_util.go

```diff
@@ -460,8 +508,15 @@ func (p *pureClient) createHost(hostName string) error {
 }
 
 // updateHost updates an existing host.
-func (p *pureClient) updateHost(hostName string) error {
-	req, err := p.createBodyReader(map[string]any{})
+func (p *pureClient) updateHost(hostName string, qns []string) error {
+	body := make(map[string]any, 1)
+
+	switch p.driver.config["pure.mode"] {
```

Comment:
should we have a default case here that returns an error if the mode is unhandled?